<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SpecialFeatureSet</class>
 <widget class="QWidget" name="SpecialFeatureSet">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>678</width>
    <height>560</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <property name="sizeConstraint">
    <enum>QLayout::SetMinAndMaxSize</enum>
   </property>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2" stretch="1,3,1">
     <property name="sizeConstraint">
      <enum>QLayout::SetMinAndMaxSize</enum>
     </property>
     <item>
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1,11,1">
       <property name="sizeConstraint">
        <enum>QLayout::SetMinAndMaxSize</enum>
       </property>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>17</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout" stretch="9,1,1">
         <property name="sizeConstraint">
          <enum>QLayout::SetMinAndMaxSize</enum>
         </property>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="10,1">
           <property name="sizeConstraint">
            <enum>QLayout::SetMinAndMaxSize</enum>
           </property>
           <item>
            <layout class="QGridLayout" name="gridLayout" columnstretch="2,0">
             <property name="sizeConstraint">
              <enum>QLayout::SetMinAndMaxSize</enum>
             </property>
             <property name="bottomMargin">
              <number>3</number>
             </property>
             <property name="horizontalSpacing">
              <number>15</number>
             </property>
             <property name="verticalSpacing">
              <number>9</number>
             </property>
             <item row="4" column="0">
              <widget class="QLabel" name="labelCardAutoApplyCharging">
               <property name="styleSheet">
                <string notr="true">font: 18pt &quot;Sans Serif&quot;;
color: rgb(255, 255, 255);</string>
               </property>
               <property name="text">
                <string>刷卡自动申请充电</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
               </property>
              </widget>
             </item>
             <item row="4" column="1">
              <widget class="QComboBox" name="comboBoxCardAutoApplyCharging">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="font">
                <font>
                 <pointsize>20</pointsize>
                </font>
               </property>
               <property name="styleSheet">
                <string notr="true">background-color: rgb(170, 170, 255);</string>
               </property>
               <item>
                <property name="text">
                 <string>是 </string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>否</string>
                </property>
               </item>
              </widget>
             </item>
             <item row="6" column="1">
              <widget class="QComboBox" name="comboBoxAbnormalElectricPowerFilter">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="font">
                <font>
                 <pointsize>20</pointsize>
                </font>
               </property>
               <property name="styleSheet">
                <string notr="true">background-color: rgb(170, 170, 255);</string>
               </property>
               <item>
                <property name="text">
                 <string>是 </string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>否</string>
                </property>
               </item>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="labelTerminalInforEndButton">
               <property name="styleSheet">
                <string notr="true">font: 18pt &quot;Sans Serif&quot;;
color: rgb(255, 255, 255);</string>
               </property>
               <property name="text">
                <string>结束充电按钮</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
               </property>
              </widget>
             </item>
             <item row="3" column="1">
              <widget class="QComboBox" name="comboBoxVINAutoCharging">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="font">
                <font>
                 <pointsize>20</pointsize>
                </font>
               </property>
               <property name="styleSheet">
                <string notr="true">background-color: rgb(170, 170, 255);</string>
               </property>
               <property name="maxVisibleItems">
                <number>10</number>
               </property>
               <item>
                <property name="text">
                 <string>是 </string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>否</string>
                </property>
               </item>
              </widget>
             </item>
             <item row="6" column="0">
              <widget class="QLabel" name="labelAbnormalElectricPowerFilter">
               <property name="styleSheet">
                <string notr="true">font: 18pt &quot;Sans Serif&quot;;
color: rgb(255, 255, 255);</string>
               </property>
               <property name="text">
                <string>异常电度数过滤</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QComboBox" name="comboBoxTerminalInforEndButton">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="font">
                <font>
                 <pointsize>20</pointsize>
                </font>
               </property>
               <property name="styleSheet">
                <string notr="true">background-color: rgb(170, 170, 255);</string>
               </property>
               <item>
                <property name="text">
                 <string>是 </string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>否</string>
                </property>
               </item>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QComboBox" name="comboBoxNetworkoutVINCharging">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <pointsize>20</pointsize>
                </font>
               </property>
               <property name="layoutDirection">
                <enum>Qt::LeftToRight</enum>
               </property>
               <property name="styleSheet">
                <string notr="true">background-color: rgb(170, 170, 255);</string>
               </property>
               <property name="maxVisibleItems">
                <number>10</number>
               </property>
               <property name="sizeAdjustPolicy">
                <enum>QComboBox::AdjustToContents</enum>
               </property>
               <item>
                <property name="text">
                 <string>是 </string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>否</string>
                </property>
               </item>
              </widget>
             </item>
             <item row="5" column="1">
              <widget class="QComboBox" name="comboBoxVINNumberOrLicenseNumber">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="font">
                <font>
                 <pointsize>20</pointsize>
                </font>
               </property>
               <property name="styleSheet">
                <string notr="true">background-color: rgb(170, 170, 255);</string>
               </property>
               <item>
                <property name="text">
                 <string>不启用</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>VIN码</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>车牌号</string>
                </property>
               </item>
              </widget>
             </item>
             <item row="2" column="0">
              <widget class="QLabel" name="labelCardTypeChoose">
               <property name="styleSheet">
                <string notr="true">font: 18pt &quot;Sans Serif&quot;;
color: rgb(255, 255, 255);</string>
               </property>
               <property name="text">
                <string>刷卡类型选择</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
               </property>
              </widget>
             </item>
             <item row="3" column="0">
              <widget class="QLabel" name="labelVINAutoCharging">
               <property name="styleSheet">
                <string notr="true">font: 18pt &quot;Sans Serif&quot;;
color: rgb(255, 255, 255);</string>
               </property>
               <property name="text">
                <string>VIN自动申请充电</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
               </property>
              </widget>
             </item>
             <item row="5" column="0">
              <widget class="QLabel" name="labelVINNumberOrLicenseNumber">
               <property name="styleSheet">
                <string notr="true">font: 18pt &quot;Sans Serif&quot;;
color: rgb(255, 255, 255);</string>
               </property>
               <property name="text">
                <string>VIN码或车牌号采集</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
               </property>
              </widget>
             </item>
             <item row="7" column="1">
              <widget class="QComboBox" name="PileTypeSetBox">
               <property name="font">
                <font>
                 <pointsize>20</pointsize>
                </font>
               </property>
               <property name="styleSheet">
                <string notr="true">background-color: rgb(170, 170, 255);</string>
               </property>
               <item>
                <property name="text">
                 <string>单桩</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>群充</string>
                </property>
               </item>
              </widget>
             </item>
             <item row="7" column="0">
              <widget class="QLabel" name="labelPileTypeSet">
               <property name="styleSheet">
                <string notr="true">font: 18pt &quot;Sans Serif&quot;;
color: rgb(255, 255, 255);</string>
               </property>
               <property name="text">
                <string>单桩/群充设置</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
               </property>
              </widget>
             </item>
             <item row="2" column="1">
              <widget class="QComboBox" name="comboBoxCardTypeChoose">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="font">
                <font>
                 <pointsize>20</pointsize>
                </font>
               </property>
               <property name="styleSheet">
                <string notr="true">background-color: rgb(170, 170, 255);</string>
               </property>
               <item>
                <property name="text">
                 <string>未选择</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>ID卡</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>特来电CPU卡</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>智能出行卡</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>招商银行卡</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>车分享卡</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>智能出行+CPU卡</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>招商银行卡+CPU卡</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>车分享卡+CPU卡</string>
                </property>
               </item>
              </widget>
             </item>
             <item row="0" column="0">
              <widget class="QLabel" name="labelNetworkoutVINCharging">
               <property name="styleSheet">
                <string notr="true">font: 18pt &quot;Sans Serif&quot;;
color: rgb(255, 255, 255);</string>
               </property>
               <property name="text">
                <string>断网VIN后六位启动充电</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
               </property>
              </widget>
             </item>
             <item row="8" column="1">
              <widget class="QComboBox" name="comboBoxPrintTicket">
               <property name="font">
                <font>
                 <pointsize>20</pointsize>
                </font>
               </property>
               <property name="styleSheet">
                <string notr="true">background-color: rgb(170, 170, 255);</string>
               </property>
               <item>
                <property name="text">
                 <string>开启</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>关闭</string>
                </property>
               </item>
              </widget>
             </item>
             <item row="8" column="0">
              <widget class="QLabel" name="labelPrintTicket">
               <property name="styleSheet">
                <string notr="true">font: 18pt &quot;Sans Serif&quot;;
color: rgb(255, 255, 255);</string>
               </property>
               <property name="text">
                <string>打印小票</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>18</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout" stretch="10,1">
           <property name="sizeConstraint">
            <enum>QLayout::SetMinAndMaxSize</enum>
           </property>
           <item>
            <spacer name="horizontalSpacer_3">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="buttonSave">
             <property name="focusPolicy">
              <enum>Qt::NoFocus</enum>
             </property>
             <property name="styleSheet">
              <string notr="true">QPushButton { background-color: rgb(0, 170, 255); color: rgb(0, 0, 0); } 
QPushButton:hover:pressed{ background-color: rgb(0,0,127); color:rgb(255, 255, 255);} 
QPushButton:hover:!pressed{ background-color: rgb(0, 170, 255); color: rgb(0, 0, 0); }
QPushButton {font: 18pt ; } </string>
             </property>
             <property name="text">
              <string>保存</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="horizontalSpacer_4">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>17</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer_3">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
